src/worker/lambda/zygote/api.go

```diff
@@ -9,5 +9,6 @@ type ZygoteProvider interface {
 	Create(childSandboxPool sandbox.SandboxPool, isLeaf bool,
 		codeDir, scratchDir string, meta *sandbox.SandboxMeta,
 		rt_type common.RuntimeType) (sandbox.Sandbox, error)
+	Warmup() error
```

Comment:
Do we need a separate call for this?  The Warmup option you added is globally visible.  Why not just check whether this is enabled in NewImportCache, and do the warmup there if so?